pycaret/containers/metrics/time_series.py

```diff
@@ -190,7 +190,7 @@ def rmsse(y_true, y_pred, y_train, sp):
     )
 
 
-def cov_prob(y_true, y_pred, lower: pd.Series, upper: pd.Series):
+def CovProb(y_true, y_pred, lower: pd.Series, upper: pd.Series):
```

Comment:
functions are still camel case. This was alright earlier (please revert it back) - you can also compare to other metrics to see what was done.